include/FilterBankDataMeta.hpp

```diff
@@ -78,7 +75,7 @@ class FilterBankDataMeta {
     size_t data_size_in_bytes() const
```

Comment:
Should we include a safeguard somewhere earlier in the pipeline along the line channel_width = -1.0 * abs(channel_width) to ensure it's always negative further along the line?

Reply:
Well, fake permits both, so by changing the sign the user would get instead of for central frequency 1400MHz (200MHz bandwidth) he would get cental frequency 1600MHz (200MHz bandwidth). Depending on the value of first channel.